lib/healthcheck/government_data_check.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Healthcheck
+  class GovernmentDataCheck
+    def initialize
+      @government_repo = BulkData::GovernmentRepository.new
+    end
+
+    def name
+      :government_data_check
+    end
+
+    def status
+      return GovukHealthcheck::CRITICAL unless @government_repo.cache_populated?
```

Comment:
Convention would be to use an attribute reader for government_repo

Reply:
I've done this, can I ask why this is convention though?

Reply:
Mostly just what GDS went with when it started, which will then be from it being a very common Ruby convention.

There's a bunch of arguments why attr_reader is the preferred approach online: https://github.com/rubocop-hq/ruby-style-guide/issues/538 (which proposed the other approach), https://ivoanjo.me/blog/2017/09/20/why-i-always-use-attr_reader-to-access-instance-variables/ but I don't think any are earth shattering reasons. For me I'm just all for consistency, which trumps my own minor preference for attr_reader.